go.mod

```diff
@@ -3,49 +3,54 @@ module github.com/kubevirt/macvtap-cni
 go 1.13
 
 require (
-	github.com/aktau/github-release v0.7.2
-	github.com/containernetworking/cni v0.7.1
+	github.com/aktau/github-release v0.8.1
+	github.com/containernetworking/cni v0.8.0
 	github.com/containernetworking/plugins v0.8.5
 	github.com/dustin/go-humanize v1.0.0 // indirect
+	github.com/github-release/github-release v0.8.1 // indirect
+	github.com/go-stack/stack v1.8.0 // indirect
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
-	github.com/kubevirt/device-plugin-manager v1.14.0
-	github.com/onsi/ginkgo v1.11.0
-	github.com/onsi/gomega v1.8.1
+	github.com/inconshreveable/log15 v0.0.0-20200109203555-b30bc20e4fd1 // indirect
+	github.com/kevinburke/rest v0.0.0-20200429221318-0d2892b400f8 // indirect
+	github.com/kubevirt/device-plugin-manager v1.18.8
+	github.com/mattn/go-colorable v0.1.7 // indirect
+	github.com/onsi/ginkgo v1.14.0
+	github.com/onsi/gomega v1.10.1
 	github.com/tomnomnom/linkheader v0.0.0-20180905144013-02ca5825eb80 // indirect
 	github.com/vishvananda/netlink v1.1.0
-	github.com/vishvananda/netns v0.0.0-20191106174202-0a2b9b5464df
+	github.com/vishvananda/netns v0.0.0-20200520041808-52d707b772fe
 	github.com/voxelbrain/goptions v0.0.0-20180630082107-58cddc247ea2 // indirect
-	golang.org/x/net v0.0.0-20190812203447-cdfb69ac37fc
-	golang.org/x/tools v0.0.0-20200117220505-0cba7a3a9ee9
-	google.golang.org/grpc v1.26.0
-	k8s.io/api v0.0.0
-	k8s.io/apimachinery v0.0.0
-	k8s.io/client-go v0.0.0
-	k8s.io/kubernetes v0.0.0-00010101000000-000000000000
+	golang.org/x/net v0.0.0-20200822124328-c89045814202
+	golang.org/x/tools v0.0.0-20200616133436-c1934b75d054
+	google.golang.org/grpc v1.31.0
+	k8s.io/api v0.19.1
+	k8s.io/apimachinery v0.19.1
+	k8s.io/client-go v0.19.1
+	k8s.io/kubelet v0.18.8
```

Comment:
Why is kubelet older than kubernetes here? And why is it required?

Reply:
deviceplugin has moved under kubelet
With v0.19.1 there is an error:
```
pkg/deviceplugin/lister.go:168:31: cannot use NewMacvtapDevicePlugin(c.Name, c.Master, c.Mode, c.Capacity, ml.NetNsPath) (type *macvtapDevicePlugin) as type dpm.PluginInterface in return argument:
	*macvtapDevicePlugin does not implement dpm.PluginInterface (missing GetPreferredAllocation method)

```
Probably related to device scheduler in 1.19.

Reply:
Puff, we may need to bump that on device-plugin-manager repo... But that can wait for a follow-up, let's not get even deeper into this. Ack, thanks